integration_test/agents/agents.go

```diff
@@ -733,8 +736,8 @@ func windowsEnvironment(environment map[string]string) string {
 // PackageLocation to determine where to install the agent from. For details
 // about PackageLocation, see the documentation for the PackageLocation struct.
 func InstallOpsAgent(ctx context.Context, logger *log.Logger, vm *gce.VM, location PackageLocation) error {
-	if location.packagesInGCS != "" && location.repoSuffix != "" {
-		return fmt.Errorf("invalid PackageLocation: cannot provide both location.packagesInGCS and location.repoSuffix. location=%#v")
+	if location.packagesInGCS != "" && location.repoSuffix != "" && location.repoURL != "" {
```

Comment:
This will only error out if all three are supplied.

Maybe change this to error out if either:

- packagesInGCS is supplied and repoSuffix is supplied (status quo)
- packagesInGCS is supplied and repoURL is supplied

i did some thinking about whether it's an error if both repoSuffix and repoURL are supplied, and long story short i don't care that much, it's fine IMO If both are supplied and repoSuffix just gets ignored.

Reply:
after an offline discussion it was decided to pass REPO_PROJECT, rather than REPO_URL, which removed the need for this change.